### DISCUSSION TITLE
Custom Theming ( Please Help )

### DISCUSSION THREAD

**Discussion by harinathpamu** [tailwindlabs/tailwindcss#3554] [Help] [answered]
Why opacities does not work on focusing with custom theming
Please help : (
Link here
https://play.tailwindcss.com/I96Gho7ie6

Thank You !

**Comment by adamwathan** [comment, marked as answer]
Hey, it's a bit more work to make CSS variable colors work with opacity. Example here: https://github.com/adamwathan/tailwind-css-variable-text-opacity-demo

**Reply by harinathpamu** [reply]
Thank You Very Much : )
Your the Super Guy
